basis_set_exchange/readers/dalton.py

```diff
@@ -95,7 +96,12 @@ def _parse_electron_lines(basis_lines, bs_data):
 
         for sh_lines in shell_blocks:
             nprim, ngen = helpers.parse_line_regex(shell_begin_re, sh_lines[0], 'nprim, ngen')
-            exponents, coefficients = helpers.parse_primitive_matrix(sh_lines[1:], nprim=nprim, ngen=ngen)
+            # fix for split over newline
+            bas_lines = sh_lines[1:]
+            if len(sh_lines[1:]) != nprim:
+                if len(sh_lines[1:]) == 2 * nprim:
+                    bas_lines = [' '.join([sh_lines[1 + i], sh_lines[2 + i]]) for i in range(nprim)]
```

Comment:
Shouldn't there be an `else` clause here..?

Reply:
Yes, this was wrong.
Now it should work for also more than a single line split (relevant for some of the Huzinaga basis sets)